packages/stream-metadata/src/routes/userRefresh.ts

```diff
@@ -57,7 +46,12 @@ export async function userRefreshOnResponse(
 			: [`/user/${userId}/*`]
 
 	try {
-		await CloudfrontManager.createCloudfrontInvalidation({ paths, logger })
+		const invalidation = await CloudfrontManager.createCloudfrontInvalidation({ paths, logger })
+		const invalidationId = invalidation?.Invalidation?.Id
+		return reply
+			.code(200)
+			.header(config.headers.invalidationId, invalidationId)
```

Comment:
@miguel-nascimento same question here: is there a reason it's on the response header and the body as well?

Reply:
@miguel-nascimento and a final comment: cf invalidations have rate limits and i'm seeing a bunch of error logs about exceeding our limit. this is why i was putting them in the onResponse hooks instead of the actual response. 

while i'm talking about error logs, @texuf the reason this error is not front and center right now is because we have a lot of error log fatigue in this service. things that are not errors (i.e stream not found) are being logged as if they're fatal errors, making it harder to understand the real errors. can we please do an error hygiene round? it's out of control

Reply:
If we need to move it to onResponse, we could use PubSub/SNS here, so we can return a quick 200 response and later the client can get the `invalidationId`. Unsure if it's the best approach to the cf invalidation rate limit.

Reply:
> is there a reason it's on the response header and the body as well?

hack to communicate between response <> onResponse hook 🙈